src/shared-components/FormFieldGroup/FormFieldGroup.tsx

```
@@ -11,13 +11,42 @@ import type { InterfaceFormFieldGroupProps } from '../../types/FormFieldGroup/in
+  hideLabel,
+}) => {
   const { t: tCommon } = useTranslation('common');
   const showError = touched && !!error;
 
+  if (inline) {
+    return (
+      <>
+        {label && !hideLabel && (
+          <Form.Label htmlFor={name} className="visually-hidden">
+            {label}
+            {required && <span aria-label={tCommon('required')}>*</span>}
```

Biome (2.1.2)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)